pandas/src/algos_groupby_helper.pxi.in

```diff
@@ -669,6 +669,72 @@ def group_cumsum(numeric[:, :] out,
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
+def group_cummin(numeric[:, :] out,
+                 numeric[:, :] values,
+                 int64_t[:] labels,
+                 numeric[:, :] accum):
+    """
+    Only transforms on axis=0
+    """
+    cdef:
+        Py_ssize_t i, j, N, K, size
+        numeric val, min_val
+        int64_t lab
+
+    N, K = (<object> values).shape
+    accum = np.empty_like(accum)
+    accum[:] = _int64_max
```

Comment:
Since val is always set before the comparison, shouldn't val == val always equal True?

```
val = values[i, j]
if val == val:
    ....
```

I notice this pattern in `group_cumsum` and `group_cumprod` as well. Should those have `nan if val != val` as well?